content/management/configuration-api/index.mdx

```diff
@@ -695,7 +704,7 @@ Here's what you need to know about **webhooks**:
 | **Chat access** | [`access_granted`](#access_granted) [`access_revoked`](#access_revoked) [`access_set`](#access_set)                                                                                                                                                                                                                                                                 |
 | **Chat users**  | [`chat_user_added`](#chat_user_added) [`chat_user_removed`](#chat_user_removed)                                                                                                                                                                                                                                                                                     |
 | **Events**      | [`incoming_event`](#incoming_event) [`event_updated`](#event_updated) [`incoming_rich_message_postback`](#incoming_rich_message_postback)                                                                                                                                                                                                                           |
-| **Properties**  | [`chat_properties_updated`](#chat_properties_updated) [`chat_properties_deleted`](#chat_properties_deleted) [`chat_thread_properties_deleted`](#chat_thread_properties_deleted) [`chat_thread_properties_updated`](#chat_thread_properties_updated) [`event_properties_updated`](#event_properties_updated) [`event_properties_deleted`](#event_properties_deleted) |
+| **Properties**  | [`chat_properties_updated`](#chat_properties_updated) [`chat_properties_deleted`](#chat_properties_deleted) [`chat_thread_properties_deleted`](#chat_thread_properties_deleted) [`chat_thread_properties_updated`](#chat_thread_properties_updated) |
```

Comment:
I have some doubts about removing this info from already published docs. Last time when we were missing implementation of some wehbooks, we added it to already frozen version 3.1. I think we should do the same here. @kogierman , @bkubiak , wdyt?

Reply:
@litwinow @bkubiak @kogierman let me know what you decide :) We can always delete them for now, and add them later when they're actually implemented.

Reply:
> > @litwinow @bkubiak @kogierman let me know what you decide :) We can always delete them for now, and add them later when they're actually implemented.
> 
> @ale-kacperczyk we've decided to keep `event_properties_updated` and `event_properties_deleted` in v3.1

Ok, @Klaym3n if you could bring those webhooks back, please.

Reply:
Done.